### PR TITLE
#182: Resize Parcels Table Column to accomodate text

### PR DESCRIPTION
**Pull request by kaydentsoftwire**
## What's changed
Adjusted the min and max widths of the table columns so that family, postcode, method, packing date and packing slot are narrower, and last status is wider, accounting for different viewing sizes

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| image image | image image |

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

### CODE REVIEW

File: package-lock.json
Comment (rickaw) [inline package-lock.json:1]: hmm what's changed in this file?

File: src/app/parcels/ParcelsPage.tsx [hunk 1]
```diff
@@ -177,26 +177,42 @@ const parcelTableColumnStyleOptions = {
...
+        basis: "5rem",
+        grow: 1,
+        minWidth: "5rem",
     },
     lastStatus: {
-        minWidth: "8rem",
+        basis: "15rem",
+        grow: 5,
     },
 };
```
[the remark this answers is not in the document]
Comment (rickaw) [reply src/app/parcels/ParcelsPage.tsx:218]: Seems like the default column styles are defined in `src/components/Tables/Table.tsx` as `defaultColumnStyleOptions` which is why grow is set to 1 by default. I'm guessing the motivation for that was probably to try and have equal width columns.

If you look closely in `src/components/Tables/Table.tsx`, we define `columns` which we pass to `DataTable` which is a library component. If you look at the type of `columns` it is `CustomColumn` which extends `TableColumn` which extends `TableColumnBase` which tells you the styles you can specify.

Maybe we should explicitly type `parcelTableColumnStyleOptions` i.e.
```ts
const parcelTableColumnStyleOptions: ColumnStyles<ParcelsTableRow> = { ... }
```

This makes me think, do we really need all those `basis`? It doesn't appear in the list of styles that can be specified.

File: package-lock.json
Comment (kaydentsoftwire) [reply package-lock.json:1]: Weird haha. This came up as a change after I pulled from dev and ran post checkout

File: src/app/parcels/ParcelsPage.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kaydentsoftwire) [reply src/app/parcels/ParcelsPage.tsx]: We do not! I have removed the unnecessary `basis`

File: package-lock.json
Comment (rickaw) [reply package-lock.json:1]: ah seems like it's changed the line endings. If you could go back to this file and change the line ending to LF and commit :) If this doesn't get resolved it's not a big deal, we can still merge it

**Review by rickaw — APPROVED** [review]

**Review by 8kristy — APPROVED** [review]

File: package-lock.json
Comment (kaydentsoftwire) [reply package-lock.json:1]: Ahh, done!

**Pull request merged**